homeassistant/components/apcupsd/binary_sensor.py

```diff
@@ -68,4 +68,8 @@ def is_on(self) -> bool | None:
         """Returns true if the UPS is online."""
         # Check if ONLINE bit is set in STATFLAG.
         key = self.entity_description.key.upper()
-        return int(self.coordinator.data[key], 16) & _VALUE_ONLINE_MASK != 0
+        # The daemon could either report just a hex ("0x05000008"), or a hex with a "Status Flag"
+        # suffix ("0x05000008 Status Flag") in older versions.
+        # Here we trim the suffix if it exists to support both.
+        flag = self.coordinator.data[key].removesuffix("Status Flag").strip()
```

Comment:
```suggestion
        flag = self.coordinator.data[key].removesuffix(" Status Flag")
```

Reply:
If this is a strict occurrence, may be we can just use `removesuffix`?

Reply:
Let's do that! PR updated :)